spec/base_spec.rb

```diff
@@ -2,169 +2,170 @@
-    it "should be comparable to other objects" do
-        base  = BaseTest.create!(name: 'joe')
-        base2 = BaseTest.create!(name: 'joe')
-        base3 = BaseTest.create!(ActiveSupport::HashWithIndifferentAccess.new(name: 'joe'))
-
-        expect(base).to eq(base)
-        expect(base).to be(base)
-        expect(base).not_to eq(base2)
-
-        same_base = BaseTest.find(base.id)
-        expect(base).to eq(same_base)
-        expect(base).not_to be(same_base)
-        expect(base2).not_to eq(same_base)
-
-        base.delete
-        base2.delete
-        base3.delete
-    end
+  it "should be comparable to other objects" do
+    base = BaseTest.create!(name: 'joe', prescribing_date: Time.now)
+    base.reload
+    base.update({ prescribing_date: '2022-07-01' })
 
-    it "should be inspectable" do
-        base = BaseTest.create!(name: 'joe')
-        expect(base.inspect).to eq("#<BaseTest id: \"#{base.id}\", name: \"joe\", job: nil>")
-    end
+  end
 
-    it "should load database responses" do
-        base = BaseTest.create!(name: 'joe')
-        resp = BaseTest.bucket.default_collection.get(base.id)
+  it "should be inspectable" do
+    base = BaseTest.create!(name: 'joe')
+    expect(base.inspect).to eq("#<BaseTest id: \"#{base.id}\", name: \"joe\", job: nil>")
+  end
 
-        base_loaded = BaseTest.new(resp, id: base.id)
+  it "should load database responses" do
+    base = BaseTest.create!(name: 'joe')
+    resp = BaseTest.bucket.default_collection.get(base.id)
 
-        expect(base_loaded.id).to eq(base.id)
-        expect(base_loaded).to     eq(base)
-        expect(base_loaded).not_to be(base)
+    base_loaded = BaseTest.new(resp, id: base.id)
 
-        base.destroy
-    end
+    expect(base_loaded.id).to eq(base.id)
+    expect(base_loaded).to eq(base)
+    expect(base_loaded).not_to be(base)
 
-    it "should not load objects if there is a type mismatch" do
-        base = BaseTest.create!(name: 'joe')
+    base.destroy
+  end
 
-        expect { CompareTest.find_by_id(base.id) }.to raise_error(CouchbaseOrm::Error::TypeMismatchError)
+  it "should not load objects if there is a type mismatch" do
+    base = BaseTest.create!(name: 'joe')
 
-        base.destroy
-    end
+    expect { CompareTest.find_by_id(base.id) }.to raise_error(CouchbaseOrm::Error::TypeMismatchError)
 
-    it "should support serialisation" do
-        base = BaseTest.create!(name: 'joe')
+    base.destroy
+  end
 
-        base_id = base.id
-        expect(base.to_json).to eq({id: base_id, name: 'joe', job: nil}.to_json)
-        expect(base.to_json(only: :name)).to eq({name: 'joe'}.to_json)
+  it "should support serialisation" do
+    base = BaseTest.create!(name: 'joe')
 
-        base.destroy
-    end
+    base_id = base.id
+    expect(base.to_json).to eq({ id: base_id, name: 'joe', job: nil, prescribing_date:nil }.to_json)
+    expect(base.to_json(only: :name)).to eq({ name: 'joe' }.to_json)
 
-    it "should support dirty attributes" do
```

Comment:
why delete test 'support dirty attributes' ?

Reply:
not deleted, just line 76 on the new file